### PR TITLE
Minor fixes on rapid7#4774

### PR DESCRIPTION
**Pull request by todb-r7**
See rapid7#4774 -- mainly, moving the module to the http scanner neighborhood.

### CODE REVIEW

**Comment by wvu-r7** [comment]
Doh! Nice catch. I was following the old pattern. :)

File: modules/auxiliary/scanner/http/chromecast_webserver.rb [hunk 1]
```diff
@@ -15,9 +15,9 @@ def initialize(info = {})
     super(update_info(info,
       'Name' => 'Chromecast Web Server Scanner',
       'Description' => %q{
-        This module scans for the Chromecast web server on port 8008/TCP.
-
-        To be used with other Chromecast modules such as chromecast_youtube.
+        This module scans for the Chromecast web server on port 8008/TCP, and
+        can be used to discover devices that can be used against Chromecast
```
Comment (wvu-r7) [inline modules/auxiliary/scanner/http/chromecast_webserver.rb]: Not sure I understand this line.

**Comment by todb-r7** [comment]
@wvu-r7 fixed

**Comment by todb-r7** [comment]
amend force push.

feel your anger

**Comment by wvu-r7** [comment]
@todb-r7:

[Let the hate flow](https://www.youtube.com/watch?v=PFkAAvDkj9k)

**Pull request merged**